velox/functions/prestosql/StringFunctions.h

```diff
@@ -430,4 +430,64 @@ struct LevenshteinDistanceFunction {
   }
 };
 
+template <typename T>
+struct NormalizeFunction {
+  VELOX_DEFINE_FUNCTION_TYPES(T);
+
+  // Map for holding normalization form options
+  static inline std::unordered_map<std::string, utf8proc_int16_t>
+      normalizatonMap{
```

Comment:
Nit : spelling normalization.

Maybe call this variable normalizationOptions ?